bundles/media.rst

```diff
@@ -0,0 +1,22 @@
+MediaBundle
+===========
+
+The MediaBundle is responsible for the handling of all media assets in Sulu.
+It also offers functionality for resizing and scaling images automatically.
+Assets can be grouped in nestable collections for better organization.
+
+Security
+--------
+
+Separate permissions can be applied to every collection. These permissions also
+apply to the assets contained in the collection.
+
+So the `view` permission is required to see the collection itself in the
+navigation and its assets in any available view in the system.
+
+The `add` permission is needed to upload new assets and add new subcollections
+to a collection. With the `edit` permission the user is allowed to edit the
+title and other attributes of the collections and its containing assets, and
+finally the `delete` permission is required to delete assets and entire
+collections.
```

Comment:
That's what I mean by assets, wrote it like this because it is not only about images (could also be documents, ...)